src/services/pasteEdits.ts

```diff
@@ -71,6 +71,7 @@ function pasteEdits(
         newText = actualPastedText ? newText.slice(0, pos) + actualPastedText[0] + newText.slice(end) : newText.slice(0, pos) + pastedText[i] + newText.slice(end);
     }
 
+    let returnImportEdits = true;

```

Comment:
Just move the `importAdder` declaration outside the `runWithTemporaryFileUpdate` callback and then you won't need a new variable